bin/src/command/orders.rs

```diff
@@ -1050,24 +1065,22 @@ impl CommandServer {
         if !self.state.handle_order(&order) {
             // Check if the backend or frontend exist before deleting it
             if worker_id.is_none() {
-                match order {
+                match &order {
                     ProxyRequestData::RemoveBackend(ref backend) => {
                         let msg = format!(
                             "cannot remove backend: application {} has no backends {} at {}",
                             backend.app_id, backend.backend_id, backend.address,
                         );
                         error!("{}", msg);
-                        self.answer_error(client_id, request_id, msg, None).await;
-                        return;
```

Comment:
the return should be kept here, to abort handling the message

Reply:
I can't retrieve this piece of code in the current state of the PR. It has been heavily refactored. For the greater good.